website/data/docs-nav-data.json

```diff
@@ -2247,6 +2247,10 @@
       {
         "title": "Key Management",
         "path": "operations/key-management"
+      },
+      {
+        "title": "Considerations for Stateful Workloads",
+        "path": "operations/stateful-workloads"
```

Comment:
Two minor thoughts on this:
* The title pushes the sidebar navigation out quite a bit. Can it be shorter?
* Does this really belong here in `operations/` rather than `concepts/` or maybe even `integrations/`?